blog/docs/releases/announcing-knative-v1-1-release.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "v1.1 release"
+linkTitle: "v1.1 release"
+Author: "[Carlos Santana](https://twitter.com/csantanapr)"
+Author handle: https://github.com/csantanapr
+date: 2021-12-14
+description: "Knative v1.1 release announcement"
+type: "blog"
+---
+
+
+### Announcing Knative v1.1 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in the documentation
+[Installing Knative](https://knative.dev/docs/admin/install/) for the respective component.
+
+#### Table of Contents
+- [Highlights](#highlights)
+- [Serving v1.1](#serving-v11)
+- [Eventing v1.1](#eventing-v11)
+- [Eventing Extensions](#eventing-extensions)
+    - [Apache Kafka Broker v1.1](#apache-kafka-broker-v11)
+    - [RabbitMQ Broker and Source v1.1](#rabbitmq-broker-and-source-v11)
+- `kn` [CLI v1.1](#client-v11)
+- [Knative Operator v1.1](#operator-v11)
+- [Thank you contributors](#thank-you-contributors)
+
+
+### Highlights
+
+- Serving now has a global `min-scale` configuration
+- Eventing brings more experimental features
+
+
+### Serving v1.1
+
+<!-- Original notes are here: https://github.com/knative/serving/releases/tag/knative-v1.1.0 -->
+
+#### 💫 New Features & Changes
```

Comment:
nit: Some of the bullet points end with periods but others don't (not sure what the official style guide recommends)

Reply:
I didn't follow a guideline, I just copy-paste the release notes verbatim from the GitHub releases pages on each repo. 😄

Reply:
went without a period